awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SharedFields.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { useField } from 'formik';
+import { FormGroup } from '@patternfly/react-core';
+import AnsibleSelect from '@components/AnsibleSelect';
+import { VariablesField } from '@components/CodeMirrorInput';
+import FormField, { CheckboxField, FieldTooltip } from '@components/FormField';
+import {
+  FormFullWidthLayout,
+  FormCheckboxLayout,
+} from '@components/FormLayout';
+
+export const SourceVarsField = withI18n()(({ i18n }) => (
+  <FormFullWidthLayout>
+    <VariablesField
+      id="source_vars"
+      name="source_vars"
+      label={i18n._(t`Environment variables`)}
+    />
+  </FormFullWidthLayout>
+));
+
+export const VerbosityField = withI18n()(({ i18n }) => {
+  const [field, meta, helpers] = useField('verbosity');
+  const isValid = !(meta.touched && meta.error);
+  const options = [
+    { value: '0', key: '0', label: i18n._(t`0 (Warning)`) },
+    { value: '1', key: '1', label: i18n._(t`1 (Info)`) },
+    { value: '2', key: '2', label: i18n._(t`2 (Debug)`) },
+  ];
+  return (
+    <FormGroup
+      fieldId="verbosity"
+      isValid={isValid}
+      label={i18n._(t`Verbosity`)}
+    >
+      <FieldTooltip
+        content={i18n._(t`Control the level of output ansible
+        will produce for inventory source update jobs.`)}
+      />
+      <AnsibleSelect
+        id="verbosity"
+        data={options}
+        {...field}
+        onChange={(event, value) => helpers.setValue(value)}
+      />
+    </FormGroup>
+  );
+});
+
+export const OptionsField = withI18n()(({ i18n }) => {
+  const [updateOnLaunchField] = useField('update_on_launch');
+  return (
+    <>
+      <FormFullWidthLayout>
+        <FormGroup
+          fieldId="option-checkboxes"
+          label={i18n._(t`Update options`)}
+        >
+          <FormCheckboxLayout>
+            <CheckboxField
+              id="overwrite"
+              name="overwrite"
+              label={i18n._(t`Overwrite`)}
+              tooltip={i18n._(t`If checked, any hosts and groups that were
+            previously present on the external source but are now removed
+            will be removed from the Tower inventory. Hosts and groups
+            that were not managed by the inventory source will be promoted
+            to the next manually created group or if there is no manually
+            created group to promote them into, they will be left in the "all"
+            default group for the inventory. When not checked, local child
```

Comment:
Old ui tool tip puts `When not checked, local child...` on a new line, like a new paragraph